build/Dockerfile

```diff
@@ -1,18 +1,16 @@
-FROM quay.io/operator-framework/ansible-operator:v0.17.0
+FROM quay.io/operator-framework/ansible-operator:v0.17.1
+USER root
 
 COPY requirements.yml ${HOME}/requirements.yml
-COPY image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
 RUN ansible-galaxy collection install -r ${HOME}/requirements.yml \
  && chmod -R ug+rwx ${HOME}/.ansible
 
-USER root
-RUN dnf install -y --nodocs redis --enablerepo=centos8-appstream && dnf clean all
-
-# Temporally workaround to fix https://github.com/operator-framework/operator-sdk/issues/2648
-RUN ln -s ${HOME}/.ansible /.ansible
+RUN rpm --nodeps -i http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-repos-8.1-1.1911.0.7.el8.i686.rpm http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-gpg-keys-8.1-1.1911.0.7.el8.noarch.rpm
```

Comment:
why are we using the i686? 

http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-repos-8.1-1.1911.0.9.el8.x86_64.rpm

Reply:
Good catch - I'll change this